DirectEmployers/MyJobs
# Tiny fix and new db.

## What's this PR do?
Adds the production API management database (confusingly named since we already have another API database, but whatever).

Adds company to the diff requirements, because as it turns out companies do in fact change.

## Where should the reviewer start?
All production settings files should have the new database.
Only code change is in import_jobs/utils.py

## How should this be manually tested?
Run an import twice. Make sure it works both times. Making sure it diffs on company name change is a lot harder (you have to change the job stored in Redis), but it's the same general concept.

## PR Checklist

### Separation of Concerns
